src/components/Reports/LocationSection.tsx

```diff
@@ -437,3 +453,27 @@ function InfoField(props: InfoFieldProps): JSX.Element {
     />
   );
 }
+
+export const buildStartedDateValid = (loc: ReportSeedBank | ReportNursery) => {
+  if (loc.buildStartedDate && loc.buildCompletedDate) {
+    return Date.parse(loc.buildStartedDate) <= Date.parse(loc.buildCompletedDate);
+  }
+  return true;
```

Comment:
should this be false? assuming we want both dates to be present and the condition to be met

Reply:
I figured `true` here since if either of the dates is not present then it'll throw a missing required field error anyways.

Reply:
That's what I figured. That's fine too. From an isolated perspective might be good to return false (assuming this code doesn't know what's happening outside of it).

Reply:
Good point. I will make that change